apps/charterafrica/src/lib/data/common/processPageTools.js

```diff
@@ -113,6 +104,9 @@ async function processPageSingleTool(page, api, context) {
         commitText: filterLabels.lastCommit,
         forksText: filterLabels.forks,
         starsText: filterLabels.stars,
+        externalLink: {
+          href: tool.docLink ?? null,
```

Comment:
Just for my understanding, where does this dockLink come from or why it doesn't behave like other links?

Reply:
@kilemensi we are reading this field from airtable (website link).
We have this field saved in cms as `link` before we added the after read hook.

Reply:
Can't we add afterRead hook now? The thing about afterRead hook is it runs when you read from the database so it doesn't change what's stored in the database.

Reply:
We have doc in the database as
```js
 {
  link: "https://datareview.openownership.org/" // From airtable
}
```

after read hook added this

```js
...
 return { ...doc, link: { href: "/tools/openownership" } };
```

Clicking the icon therefore href'ed to the page itself hence no action.

Reply:
Since this is an urgent fix, changing field in the database would maybe have required more time since involves having to wait for the cron to update the list. I will create another PR to correct this.